.github/workflows/build.yml

```diff
@@ -134,3 +134,41 @@ jobs:
       run: df -h /
       if: ${{ always() }}
 
+  AutoRust:
```

Comment:
Nit: all the other stage names are descriptive of what they do. Could this may be "code-generation" or something? Not a big deal, though.

Reply:
It is for testing AutoRust, so I just gave it `name: test AutoRust`. It does not run the AutoRust code generation.